diff --git a/node_modules/cachified/dist/common.d.ts b/node_modules/cachified/dist/common.d.ts
index 4666ac9..2e0f488 100644
--- a/node_modules/cachified/dist/common.d.ts
+++ b/node_modules/cachified/dist/common.d.ts
@@ -45,7 +45,7 @@ export interface CachifiedOptions<Value> {
      *
      * @type {Cache} Required
      */
-    cache: Cache<Value>;
+    cache: Cache<unknown>;
     /**
      * This is called when no valid value is in cache for given key.
      * Basically what we would do if we wouldn't use a cache.
